docs/activation-strategies.md

```diff
@@ -1,14 +1,48 @@
 # Activation Strategies
 
-Activation strategies is a crutial part of Unleash. 
-
-TODO! 
-- Definition lives in unleash-server
-- Implementation lives in the client implementations
-- Configuration is assiated with a feature toggle
-- Document common strategies, and how to implement them
- - gradual rollout (sticky vs. random)
- - active for user with id
- - active for host
- - active for remote host
- - totally custom
\ No newline at end of file
+It is powerful to be able to turn a feature on and off instantaneously, without redeploying the application. The next level of control comes when you are able to enable a feature for specific users or enable it for a small subset of the users. We achieve this level of control with the help of activation strategies. The simplest strategy is the “default” strategy, which basically means that the feature should be enabled to everyone.
+
+The definition of an activation strategy lives in the Unleash API and can be created via the Unleash UI. The implementation of activation strategies lives in the various client implementations. 
+
+Unleash comes with a few common activation strategies. Some of them requires the client to provide the [unleash-context](./unleash-context.md), which gives necessary context for unleash. 
+
+## default
+Is the simples activation strategies and basically means "active for everyone".
+
+## userWithId
+Active for users with a userId defined in the userIds-list. Typically I want to enable a new feature only for myself in production, before I enable it of everyone else. To achieve this we can use the “UserWithIdStrategy”. This strategy allows you to specify a list of specific user ids that you want to expose the new feature for. (A user id may of course be an email if that is more appropriate in your system.)
+
+**Parameters**
+- userIds - *List of user ids you want the feature toggle should be enabled for*
+
+## gradualRolloutUserId
+Gradually activate feature toggle for logged in users. Stickiness based on user id. ers. This strategy guarantees that the same user gets the same experience every time, across devices. It also guarantees that a user which is among the first 10% will also be among 20% roll-out degree. Thus we ensures that users gets the same experience, even if we gradually increase the number of users we expose the feature to. To achieve this we use a clever hash-algorithm where we normalize the user-id to a number between 1 and 100. 
+
+**Parameters**
+- percentage - *The percentage (0-100) you want to enable to feature toggle for.*
+- groupId - *Used to define a activation groups, which allows you to correlate across feature toggles.*
+
+## gradualRolloutSessionId
```

Comment:
"gradualRolloutUserId-strategy" => "gradualRolloutUserId strategy". And maybe backtics?